ci-operator/config/openshift/lightspeed-console/openshift-lightspeed-console-main.yaml

```diff
@@ -41,7 +40,17 @@ tests:
   steps:
     test:
     - as: lint
-      commands: yarn install && yarn run lint
+      commands: npm install && npm run lint
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: translations
```

Comment:
Minor: Can we rename this to `i18n`? Just because we're generally using that name for translation / internationalization elsewhere